app/models/miq_request.rb

```diff
@@ -559,6 +559,7 @@ def my_records
   def clean_up_keys_for_request_task
     req_task_attributes = attributes.dup
     (req_task_attributes.keys - MiqRequestTask.column_names + REQUEST_UNIQUE_KEYS).each { |key| req_task_attributes.delete(key) }
+    req_task_attributes["options"].delete(:user_message) if req_task_attributes["options"][:user_message]
```

Comment:
@tinaafitz No need for the `if` check here, just call the delete method unconditionally.